docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -89,23 +87,23 @@ starterProjects:
   name: nodejs-starter
 ```
 
-#### `commands`
-
-> The list of commands to be used when running `odo dev`.
->
-> Each command contains what will be ran within the container.
->
-> The four groups are: build, run, test and debug.
->
-> Build is what is initially ran when deploying to the cluster. It is what builds the initial container.
->
-> Run executes the command after the container has been built. 
->
-> Debug is enabled when running `odo dev --debug`.
->
-> Test executes any tests which are available and part of your application. (This is NOT-YET-IMPLEMENTED in odo)
->
-> All of these commands are needed to: build the program, execute the code, debug the application and run any applicable tests.
+### `commands`
+
+The list of commands to be used when running `odo dev`.
+
+Each command contains what will be ran within the container.
+
+The four groups are: build, run, test and debug.
+
+Build is what is initially ran when deploying to the cluster. It is what builds the initial container.
+
+Run executes the command after the container has been built. 
+
+Debug is enabled when running `odo dev --debug`.
```

Comment:
More like "used" rather than "enabled". When we say "enabled", it gives an impression that odo does something to enable it. Rather, odo just uses what's there under `debug`.